kitsune/products/jinja2/products/blocks/featured_articles_block.html

```diff
@@ -0,0 +1,18 @@
+  {% if value.featured %}
```

Comment:
similar comment as above, is this check needed?

Reply:
Dropped.